test/test_ios.py

```diff
@@ -0,0 +1,101 @@
+from __future__ import annotations
+
+import os
+import platform
+import subprocess
+
+import pytest
+
+from . import test_projects, utils
+
+basic_project = test_projects.new_c_project()
+basic_project.files["tests/__init__.py"] = ""
+basic_project.files["tests/__main__.py"] = r"""
+import platform
+import time
+
+# Workaround for CPython#130294
+for i in range(0, 10):
+    time.sleep(1)
```

Comment:
Ah, that's a bummer. Hope this can be fixed upstream soon!

Reply:
It's on my list. I've got some ideas that should be sufficient for at least a workaround.

Reply:
Actually... I think I've got a better solution that won't require any workaround at all. Fix incoming.

Reply:
(I'll still have to fix the issue upstream; it just won't be a source of a problem for this test)